octobot_commons/databases/adaptors/tinydb_adaptor.py

```diff
@@ -0,0 +1,96 @@
+#  Drakkar-Software OctoBot-Commons
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+try:
+    import tinydb
```

Comment:
Where will we install this package?

Reply:
I didnt think this through completely but for now I'm on the philosophy that we don't make is a hard requirement (same for other db drivers that might come later), however we make it a requirement to packages that use it directly (OctoBot-Trading here)